tests/test_models/test_engine/test_file_storage.py

```diff
@@ -113,3 +113,38 @@ def test_save(self):
         with open("file.json", "r") as f:
             js = f.read()
         self.assertEqual(json.loads(string), json.loads(js))
+
+    def test_get(self):
+        """Test that retrieve correctly the object requested"""
+        storage = FileStorage()
+        new_state = State("California")
+        storage.new(new_state)
+        self.assertEqual(storage.get("State", new_state.id), new_state)
+
+    def test_types_cls_id_get(self):
+        """Test the type of cls and id is str"""
+        pass
+
+    def test_get_notFound(self):
+        """Test that get doesnt found """
+        storage = FileStorage()
+        self.assertEqual(storage.get("State", "This is false"), None)
+
+    def test_count(self):
+        """Test the count of a class"""
+        self.assertEqual(models.storage.count(), len(models.storage.all()))
+
+    def test_count_class(self):
+        """Test that count the number of objects of one class"""
+        for klass in classes:
```

Comment:
Hi Camilo, I don´t understand the name 'klass' for the variable. I think we can use another meaningful name

Reply:
Changed 'klass' for 'class_in_dic'!